src/styles/blocks/content.scss

```diff
@@ -0,0 +1,8 @@
+.content {
```

Comment:
Consider renaming the class 'content' to a more descriptive name, like 'stopwatch-container'.

Reply:
I have watch-container in my block watch, but content__watch sets  external styles for positioning my clock on the page